README.md

```diff
@@ -132,3 +132,4 @@ Twitter              | Fediverse                           | Name
 @tomsontom           | @tomsontom@mastodon.social          | Tom Schindl
 @TonyPrintezis       | @TonyPrintezis@mastodon.social      | Tony Printezis
 @trisha_gee          | @trishagee@mastodon.social          | Trisha Gee
+@debugagent          | @debugagent@mastodon.social         | Shai Almog
```

Comment:
Please note that entries are sorted by Twitter handles.

Reply:
Didn't notice. Updated. Thanks!